nucypher/policy/conditions/lingo.py

```diff
@@ -697,7 +698,8 @@ def resolve_condition_class(
         conditions expression framework.
         """
         from nucypher.policy.conditions.evm import ContractCondition, RPCCondition
-        from nucypher.policy.conditions.offchain import JsonApiCondition
+        from nucypher.policy.conditions.json.api import JsonApiCondition
+        from nucypher.policy.conditions.json.rpc import JsonRpcCondition
```

Comment:
I'm wondering if we should have a higher-level module that re-exports all conditions, something like `from nucypher.policy.conditions import FooCondition, ...`. Speaking of which, I think at some moment we should drop the `policy` part from the import path. 
cc @KPrasch

Reply:
We can, but this is the Python API which we don't document / broadcast to adopters - so this would really be for our own benefit. Of course, we can reconfigure the packages in a different PR if you like.